content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -233,11 +233,14 @@ or is done when Pod is running and underlying file system supports online expans
 FlexVolumes allow resize if the driver is set with the `RequiresFSResize` capability to true. 
 The FlexVolume can be resized on pod restart. 
 
-{{< feature-state for_k8s_version="v1.11" state="alpha" >}}
-
 #### Resizing an in-use PersistentVolumeClaim
 
-Expanding in-use PVCs is a beta feature and is enabled by default via `ExpandInUsePersistentVolumes` feature gate.
+{{< feature-state for_k8s_version="v1.15" state="beta" >}}
+
+{{< note >}}
+Expanding in-use PVCs is available as beta since 1.15, and as alpha since Kubernetes 1.11. The `ExpandInUsePersistentVolumes` feature must be enabled, which is the case automatically for many clusters for beta features. Please refer to the [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) documentation for more information.
```

Comment:
> The `ExpandInUsePersistentVolumes` feature must be enabled, which is the case automatically for many clusters for beta features.

The sentence above needs some editing.

Reply:
I took it from another feature gate description - https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#publish-validation-schema-in-openapi-v2 .
> Note: OpenAPI v2 Publishing is available as beta since 1.15, and as alpha since 1.14. The CustomResourcePublishOpenAPI feature must be enabled, which is the case automatically for many clusters for beta features. Please refer to the feature gate documentation for more information.

I suggest to apply the improvements with another PR in all occurrences.

Reply:
Thanks for the explanation.